ambari-infra/ambari-infra-solr-client/src/main/resources/solrIndexHelper.sh

```diff
@@ -68,6 +68,11 @@ function upgrade_core() {
     verbose="-verbose"
   fi
 
+  if [[ -f "$INDEX_DIR/write.lock" ]]; then
```

Comment:
This seems a barbaric :)  is there is no better way for the application to delete write lock itself?

Reply:
the UpgradeIndexerTool is not deleting it, so not really, also we do not need the lock as no solr that is using the backup index